beacon-chain/core/types/state_test.go

```diff
@@ -216,71 +205,62 @@ func areBytesEqual(s1, s2 []byte) bool {
 	return true
 }
 
-func TestCalculateNewActiveState(t *testing.T) {
-	block := NewBlock(&pb.BeaconBlock{
-		Slot:           10,
-		AncestorHashes: [][]byte{{}},
-	})
-
-	cState, err := NewGenesisCrystallizedState(nil)
-	if err != nil {
-		t.Fatalf("failed to initialize genesis crystallized state: %v", err)
-	}
-
-	recentBlockHashes := [][]byte{}
-	for i := 0; i < 2*int(params.BeaconConfig().CycleLength); i++ {
-		recentBlockHashes = append(recentBlockHashes, []byte{0})
-	}
-
-	aState := NewActiveState(&pb.ActiveState{
-		PendingAttestations: []*pb.AggregatedAttestation{
-			{
-				Slot:  0,
-				Shard: 0,
-			},
-			{
-				Slot:  0,
-				Shard: 1,
-			},
-		},
-		RecentBlockHashes: recentBlockHashes,
-	})
-
-	aState, err = aState.CalculateNewActiveState(block, cState, 0)
-	if err != nil {
-		t.Fatalf("failed to calculate new active state: %v", err)
-	}
-
-	if len(aState.PendingAttestations()) != 2 {
-		t.Fatalf("expected 2 pending attestations, got %d", len(aState.PendingAttestations()))
-	}
-
-	if len(aState.RecentBlockHashes()) != 2*int(params.BeaconConfig().CycleLength) {
-		t.Fatalf("incorrect number of items in RecentBlockHashes: %d", len(aState.RecentBlockHashes()))
-	}
-
-	aState, err = aState.CalculateNewActiveState(block, cState, 0)
-	if err != nil {
-		t.Fatalf("failed to calculate new active state: %v", err)
-	}
-
-	if len(aState.PendingAttestations()) != 2 {
-		t.Fatalf("expected 2 pending attestations, got %d", len(aState.PendingAttestations()))
-	}
-
-	if len(aState.RecentBlockHashes()) != 2*int(params.BeaconConfig().CycleLength) {
-		t.Fatalf("incorrect number of items in RecentBlockHashes: %d", len(aState.RecentBlockHashes()))
-	}
-}
-
-func createHashFromByte(repeatedByte byte) []byte {
-	hash := make([]byte, 32)
-	for i := 0; i < 32; i++ {
-		hash[i] = repeatedByte
-	}
-
-	return hash
-}
+// func TestCalculateNewActiveState(t *testing.T) {
```

Comment:
Delete or uncomment. Why is this commented out?

Reply:
Deprecated by spec, will delete